ui/src/services/AxiosService/AxiosService.ts

```diff
@@ -0,0 +1,69 @@
+import { Errors, Type } from "io-ts";
+import * as E from 'fp-ts/Either';
+import { pipe } from "fp-ts/pipeable";
+import reporter from 'io-ts-reporters';
+import axios, { AxiosRequestConfig } from "axios";
+
+export async function fetchJson<T, O, I>(
```

Comment:
I'd prefer to split this method into two responsibilities. One responsible for getting the data and the second one for validating it